src/Storage/RateLimitStorageInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Bedrock\Bundle\RateLimitBundle\Storage;
+
+use Bedrock\Bundle\RateLimitBundle\Model\RateLimit;
+use Bedrock\Bundle\RateLimitBundle\Model\StoredRateLimit;
+
+interface RateLimitStorageInterface
+{
+    /**
+     * Retrieves the stored rate limit or returns null
+     */
+    public function getStoredRateLimit(RateLimit $rateLimit): ?StoredRateLimit;
+
+    /**
+     * Should store rate limit with hits = 1
```

Comment:
Why `should` ? 
```suggestion
     * Stores rate limit with hits = 1
```
Or `Initializes rate limit with hits = 1`